apps/api/src/modules/summary/summary.resolver.ts

```diff
@@ -0,0 +1,22 @@
+import { Resolver, Query, Mutation, Args } from '@nestjs/graphql';
+import { SummaryService } from './summary.service';
+import { Summary } from './entities/summary.entity';
+import { CreateSummaryDTO } from './dto/create-summary.dto';
+
+@Resolver(() => Summary)
+export class SummaryResolver {
+  constructor(private readonly summaryService: SummaryService) { }
+
+  @Mutation(() => Summary)
+  async createSummary(
+    @Args('createSummaryInput') createSummaryInput: CreateSummaryDTO,
+  ): Promise<Summary> {
+    return this.summaryService.createSummary(createSummaryInput);
+  }
+
+  @Query(() => [Summary], { name: 'summary' })
```

Comment:
Are you not creating different PR for this as this PR is for Adding File upload API.

Reply:
A valid query is needed for building the schema file so I added this function. It'll be later updated in the next PR for fetching all records.